yaksh/management/commands/create_demo_course.py

```diff
@@ -0,0 +1,77 @@
+import os
+from django.core.management.base import BaseCommand
+from yaksh.models import Course, Question, Quiz, QuestionPaper, Profile, FileUpload
+from yaksh.file_utils import extract_files
+from django.contrib.auth.models import User
+from django.utils import timezone
+from django.core.files import File
+from datetime import datetime, timedelta
+import pytz
+import zipfile
+
+
+def create_demo_course():
+    """ creates a demo course, quiz """
+
+    success = False
+    print("Creating Demo User...")
+    # create a demo user
+    user, u_status = User.objects.get_or_create(username='demo_user',
+                                                password='demo',
```

Comment:
If you create password as plain text then you won't be able to login via interface because django converts the plain password to hash. You can use set_password method.
If you are not going to use this user via interface then its fine.